### ISSUE TITLE
Compress downloaded datasets

### ISSUE THREAD

**Issue by fepegar** [fepegar/torchio#250]
Some of the available datasets are downloaded in an uncompressed format. For example, Colin27 version 2008 takes 1014 MB. Some storage could be saved if the images were compressed after downloading.

**Comment by fepegar** [comment]
Given that decompressing might take a considerable amount of time if the volume is large, and storage is getting cheaper every day, this might not be a good idea.

**Comment by fepegar** [comment]
We should check if this is worth doing or not. Storing label maps in .nii doesn't seem to be a good idea.

**Comment by romainVala** [comment]
yes, for sure label with a lot of label is increasing (linearly with the number of lable), and compression give a huge difference (and may be not so long to compute since there a lot of 0)

Actually I facing the same issue but for cpu or GPU memory, when having more than 20 label, the prediction (in float, take a lot of ram), and idem when saving those prediction on disk, It makes huge file (even compressed, because then it is in float)  but sorry, this is an other topic.

**Comment by fepegar** [comment]
Is this for multiclass voxels? If not, the images could be 3D instead.
